phalcon/http/statuscode.zep

```diff
@@ -0,0 +1,182 @@
+
+/*
+ +------------------------------------------------------------------------+
+ | Phalcon Framework                                                      |
+ +------------------------------------------------------------------------+
+ | Copyright (c) 2011-2015 Phalcon Team (http://www.phalconphp.com)       |
+ +------------------------------------------------------------------------+
+ | This source file is subject to the New BSD License that is bundled     |
+ | with this package in the file docs/LICENSE.txt.                        |
+ |                                                                        |
+ | If you did not receive a copy of the license and are unable to         |
+ | obtain it through the world-wide-web, please send an email             |
+ | to license@phalconphp.com so we can send you a copy immediately.       |
+ +------------------------------------------------------------------------+
+ | Authors: Andres Gutierrez <andres@phalconphp.com>                      |
+ |          Eduar Carvajal <eduar@phalconphp.com>                         |
+ +------------------------------------------------------------------------+
+ */
+
+namespace Phalcon\Http;
+
+/**
+ * Phalcon\Http\StatusCode
+ *
+ * Constants enumerating the HTTP status codes
+ */
+abstract class StatusCode
+{
+	// INFORMATIONAL CODES
+	const INFO_CONTINUE = 100;
+	const INFO_SWITCH_PROTOCOL = 101;
+	const INFO_PROCESSING = 102;
+
+	// SUCCESS CODES
+	const SUCCESS_CREATED = 201;
```

Comment:
If u created a class for status codes, can u please provide all codes, for example `200` too